### DISCUSSION TITLE
Blank screen after deployment, downloads it instead?

### DISCUSSION THREAD

**Discussion by Aditya-Sinha0** [vercel/community#2934] [Help] [answered]
### Summary

I'm setting up my first project off of this repo - https://github.com/Aditya-Sinha0/TimeSpentTogether, and it runs locally just fine. However, after I deployed it to vercel and pull it up(timespenttogether.vercel.app), it comes up as completely blank. I'm not really sure what to about this, and I would appreciate any help!

Thank you!

### Example

https://timespenttogether-lv9xnho40-aditya-sinha0.vercel.app

### Steps to Reproduce

pulling up this link - https://timespenttogether-lv9xnho40-aditya-sinha0.vercel.app

**Comment by smaeda-ks** [comment, marked as answer]
@Aditya-Sinha0 make sure to move your files into `/api` folder. Otherwise, they will not be made available to serverless functions:
https://vercel.com/docs/concepts/functions/serverless-functions/runtimes/python
